### PR TITLE
Adds configuration for uwsgi+nginx 

### PR DESCRIPTION
**Pull request by halfak**
Also, implements batch style scoring which should provide a substantial speed improvement.

### CODE REVIEW

File: config/ores.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  fawiki_api:
+    url: https://fa.wikipedia.org/w/api.php
+    language: persian
+  trwiki_api:
+    url: https://tr.wikipedia.org/w/api.php
+    language: turkish
+  frwiki_api:
+    url: https://fr.wikipedia.org/w/api.php
+    language: french
+  simplewiki_api:
```
Comment (he7d3r) [inline config/ores.yaml]: Should this be commented out too, as above?

File: config/ores.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    model_file: models/ptwiki.reverted.linear_svc.model
+  frwiki_revert:
+    model_file: models/frwiki.reverted.linear_svc.model
+  fawiki_revert:
+    model_file: models/fawiki.reverted.linear_svc.model
+  ptwiki_revert:
+    model_file: models/ptwiki.reverted.linear_svc.model
+  trwiki_revert:
+    model_file: models/trwiki.reverted.linear_svc.model
+  simplewiki_revert:
```
Comment (he7d3r) [inline config/ores.yaml]: same here

File: config/ores.yaml (hunk 1 above)
Comment (halfak) [inline config/ores.yaml]: Negative.  That way, once those models are ready, we only have to remove one set of comments.

**Comment by he7d3r** [comment]
Looks good.

**Pull request merged**